mllib/src/main/scala/org/apache/spark/mllib/clustering/LDAOptimizer.scala

```diff
@@ -134,8 +146,10 @@ final class EMLDAOptimizer extends LDAOptimizer {
       verticesTMP.reduceByKey(_ + _)
     }
 
+    // vertices are non empty docs and empty docs
+    val docAllTermVertices = docTermVertices union emptyDocTermVertices
```

Comment:
I'm not 100% sure this is a hard style rule, but best to avoid this syntax and write `a.union(b)`. It generates a compiler warning otherwise IIRC.